src/Model/Resolver/Products/DataPostProcessor/Attributes.php

```diff
@@ -209,41 +207,44 @@ protected function appendWithOptions(
 
```

Comment:
can you check if $detailedAttributes actually needs to be reloaded from $attributeRepository?
I highly doubt it is necessary / provided $attributes may already have all the necessary data in attribute_code => attribute model format